.gitignore

```diff
@@ -40,3 +41,6 @@ sbt_cache/*
 **/.ammonite/
 **/metals.sbt
 **/pom.xml
+universal-application-tool-0.0.1/package-lock.json
+universal-application-tool-0.0.1/pom.bkp
+universal-application-tool-0.0.1/.sbtopts
```

Comment:
issue (blocking): these gitignore changes are unrelated to the rest of the PR, would prefer to have it separate